src/material/form-field/form-field.ts

```diff
@@ -284,14 +281,19 @@ export class MatFormField extends _MatFormFieldMixinBase
 
   constructor(
       public _elementRef: ElementRef, private _changeDetectorRef: ChangeDetectorRef,
-      @Optional() @Inject(MAT_LABEL_GLOBAL_OPTIONS) labelOptions: LabelOptions,
+      /**
+       * @deprected `_labelOptions` parameter no longer being used. To be removed.
```

Comment:
It's a bit unfortunate that we didn't deprecate this parameter (my bad obviously!)

```suggestion
       * @deprecated `_labelOptions` parameter no longer being used. To be removed.
```